yt_dlp/extractor/dumpert.py

```diff
@@ -10,7 +10,7 @@ class DumpertIE(InfoExtractor):
     _VALID_URL = r'''(?x)
         (?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl(?:
             /(?:mediabase|embed|item)/|
-            (?:/toppers|/latest|/?)\?selectedId=
+            [^#]+[?&]selectedId=
```

Comment:
and if we apply my above suggestion then we'll need to do this too
```suggestion
            [^#]*[?&]selectedId=
```